mm5176.md

```diff
@@ -6,9 +6,14 @@ UNI: mm5176
 
 Github: [link](https://github.com/mikaylamancini1)
 
+GitHub README: [link](https://github.com/mikaylamancini1/mikaylamancini1/blob/main/README.md)
+
+
 
 ## Project Proposal
 I am interested in building some type of web scraper for scores of sports games and comparing that data to what the sportsbetting odds and outcomes were.
+- [sportbetting]([./projects/javascript/temporal-cache.md](https://github.com/mikaylamancini1/project-proposals-s2023/blob/main/projects/javascript/sportsbetting.md))
 
 ## Project Contributions
 Numpy and Matplotlib have a lot of Good First Issues tagged and I am also interested in Jupyter Notebook.
+- [Numpy](https://github.com/mikaylamancini1/project-proposals-s2023/blob/main/projects/python/numpy.md)
```

Comment:
still needs to be a relative link like `./projects/python/numpy.md`, otherwise it will link to your fork instead of staying on this repository.